packages/angular_devkit/build_angular/test/tslint/works_spec_large.ts

```diff
@@ -25,6 +25,22 @@ describe('Tslint Target', () => {
     ).toPromise().then(done, done.fail);
   }, 30000);
 
+  it('should report lint error once', (done) => {
+    host.writeMultipleFiles({'src/app/app.component.ts': 'const foo = "";\n' });
+    const logger = new TestLogger('lint-error');
+
+    runTargetSpec(host, tslintTargetSpec, undefined, DefaultTimeout, logger).pipe(
+      tap((buildEvent) => expect(buildEvent.success).toBe(false)),
+      tap(() => {
+        // this is to make sure there are no duplicates
```

Comment:
This is due that duplicate messages are within the same entry since under the hood we are using tslint formatter to format the messages with will convert and array of results to a string